app/views/gallery/index.html.erb

```diff
@@ -1,6 +1,13 @@
 <%= title(t('gallery.title')) %>
 <div class="headline headline-md">
-  <h1><%= t('gallery.title') %></h1>
+  <h1>
+    <%= t('gallery.title') %> -
+    <% if @year.present? %>
```

Comment:
Also, you already do this check here:
https://github.com/fsek/web/blob/ef948d9606702ba3d1310e137976525f095865bb/app/controllers/gallery_controller.rb#L19

This is too confusing and should be cleaned up. I recommend you (1) don't have both @year and @years as instance vars, it's just confusing (2) do all the checks in one place, preferably the controller.